apps/mobile/src/features/balances/assets/assets-list.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactElement, useMemo } from 'react';
+
+import { Screen } from '@/components/screen/screen';
+import { sortSip10Balances } from '@/features/balances/utils/sort-sip10-balances';
+import { useRunesFlag } from '@/features/feature-flags';
+import { RefreshControl } from '@/features/refresh-control/refresh-control';
+import { useRunesAccountBalance } from '@/queries/balance/runes-balance.query';
+import { useSip10AccountBalance } from '@/queries/balance/sip10-balance.query';
+
+import { RuneBalance, Sip10Balance } from '@leather.io/services';
+
+import { renderAsset } from './render-assets';
+
+interface AssetsListProps {
+  sip10Data: ReturnType<typeof useSip10AccountBalance>;
+  runesData: ReturnType<typeof useRunesAccountBalance>;
+  header: ReactElement;
+  onPressToken?: (tokenId: string) => void;
+}
+
+export function AssetsList({ sip10Data, runesData, header, onPressToken }: AssetsListProps) {
+  const runesFlag = useRunesFlag();
+
+  const sip10Memo = useMemo(() => {
+    if (sip10Data.state === 'success') return sip10Data.value.sip10s.sort(sortSip10Balances);
+    return [];
+  }, [sip10Data]);
+
+  // TODO: No computation, memo is unnecessary. Tuck all of this into a hook?
+  const runesMemo = useMemo(() => {
+    if (runesData.state === 'success' && runesFlag) return runesData.value.runes;
+    return [];
+  }, [runesData, runesFlag]);
+
+  // TODO: Unnecessary memoization
+  const allAssetsMemo = useMemo(() => [...sip10Memo, ...runesMemo], [sip10Memo, runesMemo]);
+
+  return (
+    // TODO: Redundant fragment
+    <>
+      {/*TODO: Generic isn't necessary*/}
+      <Screen.FlashList<Sip10Balance | RuneBalance>
+        data={allAssetsMemo}
+        renderItem={({ item }) =>
+          renderAsset({
+            item,
+            onPress: () => {
+              if (item.asset.protocol === 'sip10') onPressToken?.(item.asset.assetId);
+            },
+          })
+        }
+        getItemType={item => item.asset.protocol}
+        showsVerticalScrollIndicator={false} // TODO: Reason?
```

Comment:
Maybe we don't need this, lets try removing. I think I added it to stop the scroll bar showing but that was before we moved the top content to `ListHeader` so it could be OK to remove